### ISSUE TITLE
Rain on windshield moves at to low of a windspeed

### ISSUE THREAD

**Issue by wlbragg** [c172p-team/c172p#1251]
I noticed while working on the J3Cub the animated rain effect on the windshield moves at too low of a airspeed. The c172p uses the same model. It was set at 5 and I increased it to 20 or greater.

The way it currently works is airspeed is the driver until there is a engine RPM. The idea being we should be seeing only drops until there is either a high enough airspeed pushing the rain across the windshield or the prop is turning.

The value of 20 for airspeed is closer to what I would expect it would take to push wind across the windshield, but it may need to be even higher.

**Comment by wlbragg** [comment]
Yes but then it would require direction as well. The property being used is

>/velocities/airspeed-kt

As that property is the one we use for stresses like vne, it should be a "close enough" indicator of the wind blowing over the aircraft from front to back, without having to do any other calculations.

**Comment by wkitty42** [comment]
wind direction should be available as well... it is used to push you off course when flying perpendicular to its direction of travel... i think... i may have been seeing drift from the compass, though, way back when i was doing my (impossible?) long flights in the c172p from KSFO non-stop to (eg) Denver or Groom Lake using the AP and heading bug for direction control... i know CA craft access and display the weather stats on their displays so at least the METAR data should be available for us to use in our animations like this... the FG-1000 may display the weather stats, too...

i really like what you've done, though... especially from before with the rain moving depending on the prop wash... adding in the rain moving properly when the engine is idling or stopped is another nice realism touch :)


NOTE: "weather stats" like wind direction and speed, pressure, pressure direction trend (rising/falling), etc...

**Comment by gilbertohasnofb** [comment]
> The way it currently works is windspeed is the driver until there is a engine RPM. 

Wouldn't using airspeed be better than windspeed? After all, airspeed is the speed that the wind speed in the component pointing forward in relation to the aircraft regardless if the aircraft is in movement or not.

**Comment by gilbertohasnofb** [comment]
@wlbragg Great. So is that PR ready to be merged?

**Comment by wlbragg** [comment]
Yes

**Comment by gilbertohasnofb** [comment]
@wlbragg Looks good so I merged it. I did find a weird thing related to the rain effect but which is not caused by your PR. The side windows display rain substantially different than the sides of the front glass, compare:

![image](https://user-images.githubusercontent.com/5700795/51092338-ee754080-178d-11e9-81f0-6d12361b22ee.png)

Is this worth investigating in your opinion or is this just a consequence of how these are mapped into the windows?
